focuspoints.lrdevplugin/Utils.lua

```diff
@@ -46,18 +46,58 @@ function splitText(str, delim)
   return r
 end
 
+function split(str, delim)
```

Comment:
that's good ! docing is not my force. And none of the other functions there were doced. I used a breach so to say :-)